src/Sulu/Component/Rest/FlattenExceptionNormalizer.php

```diff
@@ -42,6 +42,13 @@ public function __construct(
         $this->translator = $translator;
     }
 
+    public function getSupportedTypes(?string $format): array
+    {
+        return [
+            FlattenException::class => false,
+        ];
```

Comment:
I see but as this is a decorated service I think we should in this case just call:

```suggestion
        return $this->decoratedNormalizer->getSupportedTypes($format);
```

Reply:
I think the result would be currently the same, but the `supportsNormalization` method does not call the decorated service, so in my opinion it shouldn't do that in the `getSupportedTypes` method either.

getSupportedTypes specifies the cacheability of the result of its own supportsNormalization.

If one day `FOS\RestBundle\Serializer\Normalizer\FlattenExceptionNormalizer::getSupportedTypes` will return `true`, Sulus' `getSupportedTypes` still should return `false`, because the result of its own `supportsNormalization` still must not be cached.

I hope you get my point :-)

Reply:
but its not a `normalizer` by its own its a decorated service and should behave like the decorate normalizer.

Reply:
Why does `supportsNormalization` not call the decorated service like this

```php
    public function supportsNormalization(mixed $data, ?string $format = null, array $context = []): bool
    {
        return $this->decoratedNormalizer->supportsNormalization($data, $format, $context);
    }
```

Reply:
I remember now because the older supported version didn't have that method. So the PR is fine this way maybe in future we can update it to use `parent::` for both methods.